modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,568 @@
+```
+
+[[mounted-disks]]
+=== Confirm disks are correctly mounted
+
+Ensure that the correct disks are mounted and if multiple devices are used, they are configured as RAID-0. Other RAID configurations can have significantly worse latencies. The file system should be of type XFS. If XFS is unavailable, ext4 is an appropriate alternative. 
+
+Example:
+```bash
+grep data_directory /etc/redpanda/redpanda.yaml
+    data_directory: /var/lib/redpanda/data
+df -khT /var/lib/redpanda/data
+```
+
+Output:
+```bash
+Filesystem     Type  Size  Used Avail Use% Mounted on
+/dev/nvme0n1   xfs   1.8T   14G  1.8T   1% /mnt/vectorized
+```
+
+Output:
+
+```bash
+Filesystem     Type  Size  Used Avail Use% Mounted on
+/dev/md0       xfs    14T   99G   14T   1% /mnt/vectorized
+```
+
+Output:
+```bash
+mdadm --detail /dev/md0
+```
+
+Output:
+```bash
+/dev/md0:
+           Version : 1.2
+     Creation Time : Thu Apr 18 11:03:46 2024
+        Raid Level : raid0
+        Array Size : 14648172544 (13969.59 GiB 14999.73 GB)
+      Raid Devices : 2
+     Total Devices : 2
+       Persistence : Superblock is persistent
+
+       Update Time : Thu Apr 18 11:03:46 2024
+             State : clean
+    Active Devices : 2
+   Working Devices : 2
+    Failed Devices : 0
+     Spare Devices : 0
+
+            Layout : -unknown-
+        Chunk Size : 512K
+
+Consistency Policy : none
+
+              Name : ip-172-31-24-82:0  (local to host ip-172-31-24-82)
+              UUID : e9574118:10d562bf:ed3ca2d9:68ccc3a6
+            Events : 0
+
+    Number   Major   Minor   RaidDevice State
+       0     259        2        0      active sync   /dev/nvme2n1
+       1     259        0        1      active sync   /dev/nvme1n1
+```
+
+[[auth-enable]]
+=== Confirm authentication is enabled
+
+Ensure that xref:manage:security/authentication.adoc[authentication] is set up (or other mitigations are in place). Without SASL authentication enabled, anybody can potentially connect to the Redpanda brokers.
+
+Example:
+```bash
+rpk cluster config get kafka_enable_authorization
+```
+
+Output:
+```bash
+true
+```
+
+[[super-users]]
+=== Confirm that superusers are configured
+
+Ensure that the xref:deploy:deployment-option/self-hosted/manual/production/production-deployment.adoc#secure-the-admin-api[Admin API is secured], and any users defined in the superusers configuration are appropriately protected with strong credentials.
+
+See also:
+xref:manage:security/authentication.adoc#create-superusers[Create superusers]
+
+[[tls-enabled]]
+=== TLS Enabled 
+
+Ensure that all public interfaces have xref:manage:security/encryption.adoc[TLS enabled].
+
+Example:
+```bash
+grep tls /etc/redpanda/redpanda.yaml
+    kafka_api_tls:
+    admin_api_tls:
+    rpc_server_tls:
+        tls:
+        tls:
+    schema_registry_api_tls:
```

Comment:
```suggestion
journalctl -u redpanda.service | grep tls
Jun 06 12:41:35 ip-172-31-31-199 rpk[9673]: INFO  2024-06-06 12:41:35,513 [shard 0:main] main - application.cc:772 - redpanda.cloud_storage_disable_tls:0        - Disable TLS for all S3 connections
Jun 06 12:41:35 ip-172-31-31-199 rpk[9673]: INFO  2024-06-06 12:41:35,514 [shard 0:main] main - application.cc:772 - redpanda.kafka_mtls_principal_mapping_rules:{nullopt}        - Principal Mapping Rules for mTLS Authentication on the Kafka API
Jun 06 12:41:35 ip-172-31-31-199 rpk[9673]: INFO  2024-06-06 12:41:35,514 [shard 0:main] main - application.cc:772 - **redpanda.admin_api_tls:{{name: , tls_config: { enabled: 1** key/cert files: {{ key_file: /etc/redpanda/certs/node.key cert_file: /etc/redpanda/certs/node.crt }} ca file: {/etc/redpanda/certs/truststore.pem} client_auth_required: 0 }}}        - TLS configuration for admin HTTP server
Jun 06 12:41:35 ip-172-31-31-199 rpk[9673]: INFO  2024-06-06 12:41:35,515 [shard 0:main] main - application.cc:772 - **redpanda.kafka_api_tls:{{name: , tls_config: { enabled: 1** key/cert files: {{ key_file: /etc/redpanda/certs/node.key cert_file: /etc/redpanda/certs/node.crt }} ca file: {/etc/redpanda/certs/truststore.pem} client_auth_required: 0 }}}        - TLS configuration for Kafka API endpoint
Jun 06 12:41:35 ip-172-31-31-199 rpk[9673]: INFO  2024-06-06 12:41:35,515 [shard 0:main] main - application.cc:772 - **redpanda.rpc_server_tls:{ enabled: 1** key/cert files: {{ key_file: /etc/redpanda/certs/node.key cert_file: /etc/redpanda/certs/node.crt }} ca file: {/etc/redpanda/certs/truststore.pem} client_auth_required: 0 }        - TLS configuration for RPC server
Jun 06 12:41:35 ip-172-31-31-199 rpk[9673]: INFO  2024-06-06 12:41:35,515 [shard 0:main] main - application.cc:772 - pandaproxy.pandaproxy_api_tls:{}        - TLS configuration for Pandaproxy api
Jun 06 12:41:35 ip-172-31-31-199 rpk[9673]: INFO  2024-06-06 12:41:35,515 [shard 0:main] main - application.cc:772 - **pandaproxy_client.broker_tls:{ enabled: 1** key/cert files: {{ key_file: /etc/redpanda/certs/node.key cert_file: /etc/redpanda/certs/node.crt }} ca file: {/etc/redpanda/certs/truststore.pem} client_auth_required: 0 }        - TLS configuration for the brokers
Jun 06 12:41:35 ip-172-31-31-199 rpk[9673]: INFO  2024-06-06 12:41:35,515 [shard 0:main] main - application.cc:772 - **schema_registry.schema_registry_api_tls:{{name: , tls_config: { enabled: 1** key/cert files: {{ key_file: /etc/redpanda/certs/node.key cert_file: /etc/redpanda/certs/node.crt }} ca file: {/etc/redpanda/certs/truststore.pem} client_auth_required: 0 }}}        - TLS configuration for Schema Registry API
Jun 06 12:41:35 ip-172-31-31-199 rpk[9673]: INFO  2024-06-06 12:41:35,515 [shard 0:main] main - application.cc:772 - **schema_registry_client.broker_tls:{ enabled: 1** key/cert files: {{ key_file: /etc/redpanda/certs/node.key cert_file: /etc/redpanda/certs/node.crt }} ca file: {/etc/redpanda/certs/truststore.pem} client_auth_required: 0 }        - TLS configuration for the brokers
Jun 06 12:41:35 ip-172-31-31-199 rpk[9673]: INFO  2024-06-06 12:41:35,515 [shard 0:main] main - application.cc:772 - audit_log_client.broker_tls:{ enabled: 1 key/cert files: {{ key_file: /etc/redpanda/certs/node.key cert_file: /etc/redpanda/certs/node.crt }} ca file: {/etc/redpanda/certs/truststore.pem} client_auth_required: 0 }        - TLS configuration for the brokers
```
Using the logs on each broker we can check that the following interfaces have TLS enabled:
- Kafka API
- Admin REST API
- Internal RPC Server
- Schema Registry
- Pandaproxy
In the logs, check for `enabled: 1`